Makefile

```diff
@@ -42,12 +42,20 @@ ifdef HTTPS_PROXY
 	DOCKERARGS += --build-arg https_proxy=$(HTTPS_PROXY)
 endif
 
-LDFLAGS=
 ifdef STATIC
-	export CGO_ENABLED=0
-	LDFLAGS=-a -ldflags '-extldflags \"-static\"'
+    ifndef FIPS_COMPLIANT
+        export CGO_ENABLED=0
+        EXTLDFLAGS=-extldflags "-static"
+    else
+        $(error STATIC and FIPS_COMPLIANT cannot be set simultaneously)
+    endif
+else
+    export CGO_ENABLED=1
+    EXTLDFLAGS=-extldflags ""
```

Comment:
there is no need you can just have a default for `LDFLAGS` and override it when needed